tests/xtro-sharpie/iOS-CarPlay.ignore

```diff
@@ -1,2 +1,4 @@
 !incorrect-protocol-member! CPBarButtonProviding::backButton is REQUIRED and should be abstract
 !incorrect-protocol-member! CPBarButtonProviding::setBackButton: is REQUIRED and should be abstract
+!missing-selector! CPWindow::setTemplateApplicationScene: not bound
+!missing-selector! CPWindow::templateApplicationScene not bound
```

Comment:
you can add comments, any line that starts with `#`, to the internal bug you filed
not critical since `git blame` can find the commit/PR and that will link with it...

Reply:
Added the issue # as a comment here too...let me know if the formatting is correct!

Reply:
Same in your comment here, please use the full link: https://github.com/xamarin/maccore/issues/1822